### ISSUE TITLE
Multiple profiles processed for single run

### ISSUE THREAD

**Issue by robzr** [Nike-Inc/gimme-aws-creds#86]
In DSM we are using up to 5 different AWS accounts, and some Terraform configurations access multiple accounts; currently this means we have to run gimme-aws-creds once for each profile.

Would it be possible to add support for a single Okta auth to populate multiple profiles?

**Comment by jralmaraz** [comment]
We have the same need. Was this enhancement taken forward ?
 
Thanks!

**Comment by SirFerdek** [comment]
Is this the same as #145 to be closed by PR #157 ? :)

**Comment by jdfalk** [comment]
I don't think so. Having multiple profiles is nice but what I think we want is to be able to authenticate to all of the accounts at once. We have multiple AWS accounts under one Okta page, and I setup multiple profiles, one for each account as I am using `appurl` but that still means I have to individually get credentials for each account which means upwards of 8-12 MFA prompts. Maybe I'm doing something wrong but is there some way to get all of them at once?

**Comment by keyset** [comment]
@epierce I see you closed this as Completed, but I don't see in the documentation or the code where multiple `profile` names can be authenticated simultaneously. Am I missing something?

**Comment by epierce** [comment]
sorry - this was closed by mistake.
